index.html

```diff
@@ -4,9 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="./styles/index.css" rel="stylesheet" type="text/css">
+	<script src="./src/index.js"></script>
+	<script src="./node_modules/axios/dist/axios.min.js"></script>
     <title>Weather Report</title>
 </head>
-<body>
-    
+<body id="body" onload="addListeners(); updateLocation();">
```

Comment:
While this works, we can ensure that listeners load appropriately inside your `script` file instead of in the HTML